python/src/functions/create_archive.py

```diff
@@ -0,0 +1,91 @@
+import os
+import tempfile
+from typing import Any
+import zipfile
+
+import boto3
+from aws_lambda_typing.context import Context
+from mypy_boto3_s3.client import S3Client
+from src.lib.logging import get_logger, reset_contextvars
+
+S3_BUCKET = os.environ.get("S3_BUCKET", "test_bucket")
```

Comment:
confirming that the name of the bucket here will be as `os.environ.get("REPORTING_DATA_BUCKET_NAME", "test_bucket")

Reply:
Done.